block-node/stream-publisher/src/test/java/org/hiero/block/node/stream/publisher/fixtures/TestStreamPublisherManager.java

```diff
@@ -75,6 +75,11 @@ public long getLatestBlockNumber() {
         return latestBlockNumber;
     }
 
+    @Override
+    public void notifyTooFarBehind(long newestKnownBlockNumber) {
+        // Do nothing.
```

Comment:
Throwing is better, otherwise false positives might occur. We must be very explicit with fixtures


```suggestion
        throw new UnsupportedOperationException("implement notifyTooFarBehind in fixture class");
```

Reply:
P.S. feel free to not do this change, just pointing out.

Reply:
please do add it to your follow up, I agree is better to throw thought. 👍